src/App/Datums.cpp

```diff
@@ -33,11 +33,9 @@
 
 #include "Datums.h"
 
-#ifndef M_PI
-#define M_PI 3.14159265358979323846
-#endif
 
 using namespace App;
+using namespace std::numbers;
```

Comment:
Using `using namespace std::X` (similar to `using namespace std`) gets a lot more than just what is used here.
Would it be possible to change it to specifically using the constants we have in the file, to avoid potential collisions?

```suggestion
using pi = std::numbers::pi;
```